provider-example/Cargo.toml

```diff
@@ -11,11 +11,11 @@ chacha20poly1305 = { version = "0.10.0", default-features = false, features = ["
 der = "0.7.0"
 ecdsa = "0.16.8"
 hmac = "0.12.0"
-hpke-rs = { version = "0.1.2", optional = true }
```

Comment:
It would be good to move this commit up above "introduce a "std" Cargo feature", which should remove some of the back-and-forth

Reply:
done

Reply:
great, could you edit this part of the commit message? 

> hpke is gated behind the feature because as of v0.1.2 it does not have
> no-std support

Reply:
:+1: I removed it